tests/src/assertions/testElementNotPresent.js

```diff
@@ -24,7 +24,36 @@ module.exports = {
       },
       assertion : function(passed, result, expected, msg, abortOnFailure) {
         test.equals(passed, true);
-        test.equals(result, null);
+        test.equals(result, 'not present');
+        test.equals(expected, 'not present');
+        test.equals(msg, 'Testing if element <.test_element> is not present.');
+        test.equals(abortOnFailure, true);
+        test.done();
+      }
+    };
+    Api.init(client);
+    var m = Api.createAssertion('elementNotPresent', assertionFn, true, client);
+    m._commandFn('.test_element');
+  },
+
+  'elementNotPresent assertion passed when exceptions are passsed' : function(test) {
+    var assertionFn = require('../../../'+BASE_PATH + '/api/assertions/elementNotPresent.js');
+    var client = {
+      options : {},
+      locateStrategy : 'css selector',
+      api : {
+        elements : function(using, selector, callback) {
+          test.equals(selector, '.test_element');
+          test.equals(using, 'css selector');
+          callback({
+            status : -1,
+            value : (new Error())
```

Comment:
are we talking about the `(new Error())` becoming `JSON.stringify((new Error()),null,2)`?
this is essentially just `{}`...